### ISSUE TITLE
Kubescaler setup failed

### ISSUE THREAD

**Issue by gerijani** [kubecost/cost-analyzer-helm-chart#2430]
Hi,

Using HELM chart 1.103.3, installed in AKS environment. Starting kubecost-cluster-controller failing with 
←[36merror=←[0m←[31m"A cost-model HTTP base path is required. Set with CC_KUBESCALER_COST_MODEL_PATH Example: CC_KUBESCALER_COST_MODEL_PATH=http://localhost:9090/model

I found no reference to this environment variable on the documentation site.

**Comment by kwombach12** [comment]
@thomasvn Is this something you can take a look at?

**Comment by thomasvn** [comment]
@gerijani Can you provide the command you used to install Kubecost's cluster controller? This environment variable should have been created by the Helm template:

https://github.com/kubecost/cost-analyzer-helm-chart/blob/73ed26ce64fcb8553e99b0ff135698c82389d536/cost-analyzer/templates/kubecost-cluster-controller-template.yaml#L219-L220

**Comment by gerijani** [comment]
@thomasvn I've used helm upgrade, but now as I see what is this actually I've checked our chart and seems it's maybe some version problem. Though we use v1.103.3, these section is different:

- name: CLUSTER_ID
  value: {{ .Values.prometheus.server.global.external_labels.cluster_id }}
- name: TURNDOWN_NAMESPACE
  value: {{ .Release.Namespace }}
- name: TURNDOWN_DEPLOYMENT
  value: {{ template "kubecost.clusterControllerName" . }}
        {{- if .Values.kubecostProductConfigs.gcpSecretName }}
- name: GOOGLE_APPLICATION_CREDENTIALS
   value: /var/keys/service-key.json
        {{- end }}
        
So I've updated the chart and now it's OK.
Thanks a lot, it was helpful!
